firebase-appdistribution-api/CHANGELOG.md

```diff
@@ -1 +1,56 @@
 # Unreleased
```

Comment:
How does this file work?

Reply:
We will need to update this file as we merge changes into the `master` branch (see go/firebase-android/docs/workflow#release-notes-aka-changelog). To avoid having to retroactively write these we should probably start adding to this as we merge into our feature branches as well, even if the presubmit check doesn't run on those PRs.